tests/testthat/test-00-Id.R

```diff
@@ -1,14 +1,26 @@
 test_that("Id() requires a character vector", {
-  expect_snapshot(Id(1), error = TRUE)
+  expect_error(Id(1))
```

Comment:
What's wrong with the snapshot test?

Reply:
Sorry. New to this. There's nothing wrong with snapshot. I got an error because I was testing interactively in the console. Reverted to the snapshot.